src/The_Cartridge_Header.md

```diff
@@ -1,65 +1,74 @@
 # The Cartridge Header
 
-An internal information area is located at $0100-014F in each cartridge.
-It contains the following values:
+Each cartridge contains a header, located at the address range `$0100`-`$014F`.
+The cartridge header provides the following information about the game itself and the hardware it expects to run on:
 
-## 0100-0103 - Entry Point
+## 0100-0103 — Entry Point
 
-After displaying the Nintendo Logo, the built-in boot procedure jumps to
-this address ($0100), which should then jump to the actual main program
-in the cartridge. Usually this 4 byte area contains a `nop` instruction,
-followed by a `jp $0150` instruction. But not always.
+After displaying the Nintendo logo, the built-in [boot ROM](<#Power-Up Sequence>) jumps to address `$0100`, which should then jump to the actual main program in the cartridge.
+Most commercial games fill this 4-byte area with a [`nop` instruction](https://rgbds.gbdev.io/docs/v0.5.2/gbz80.7/#NOP) followed by a [`jp $0150`](https://rgbds.gbdev.io/docs/v0.5.2/gbz80.7/#JP_n16).
 
-## 0104-0133 - Nintendo Logo
+## 0104-0133 — Nintendo Logo
+
+This area contains a bitmap that is displayed when the Game Boy is powered on.
+They must match the following (hexadecimal) dump, otherwise [the boot ROM](<#Power-Up Sequence>) won't allow the game to run:
```

Comment:
Leftover from when "This area" was "These bytes".
```suggestion
It must match the following (hexadecimal) dump, otherwise [the boot ROM](<#Power-Up Sequence>) won't allow the game to run:
```